src/OmnichannelChatSDK.ts

```diff
@@ -419,12 +420,22 @@ class OmnichannelChatSDK {
 
         sessionInitOptionalParams = this.populateInitChatOptionalParam(sessionInitOptionalParams, optionalParams);
 
+        if (optionalParams.isProactiveChat) {
+            sessionInitOptionalParams.initContext!.isProactiveChat = optionalParams.isProactiveChat;
+        }
+
         if (this.isPersistentChat && !this.chatSDKConfig.persistentChat?.disable) {
             sessionInitOptionalParams.reconnectId = this.reconnectId as string;
         } else if (this.isChatReconnect && !this.chatSDKConfig.chatReconnect?.disable && !this.isPersistentChat) {
             sessionInitOptionalParams.reconnectId = this.reconnectId as string;
         }
         
+        if (this.liveChatConfig.LiveWSAndLiveChatEngJoin.msdyn_requestvisitorlocation === "true") {
```

Comment:
We should add a platform check as well. It should only be applicable on browsers only

```
if (platform.isNode() || platform.isReactNative())
```